### DISCUSSION TITLE
Ship Icon

### DISCUSSION THREAD

**Discussion by seydieser** [twbs/bootstrap#40669] [Ideas]
Hi everyone. I use bootstrap icons. They are very useful and practical. But i can not find a "Cruise Ship" icon or something related with sea, ship, boat, captain, etc. Who can design them? Thamk you all.

**Comment by julien-deramond** [comment]
Hello @seydieser 
I noticed that you've already created https://github.com/twbs/icons/issues/2014. This is the correct approach :)
The designer managing the Bootstrap Icons library reviews all icon requests regularly.
